homeassistant/components/epsonworkforce/sensor.py

```diff
@@ -10,11 +10,12 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity import Entity
 
-REQUIREMENTS = ['epsonprinter==0.0.8']
+REQUIREMENTS = ['epsonprinter==0.0.9']
 
 _LOGGER = logging.getLogger(__name__)
 MONITORED_CONDITIONS = {
     'black': ['Inklevel Black', '%', 'mdi:water'],
+    'photoblack': ['Inklevel Photoblack', '%', 'mdi:water'],
```

Comment:
Just noticed this but `inklevel` seems to be a typo shouldn't it be `ink level`?

Reply:
Yes, you are correct